src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -154,3 +172,91 @@ export function saveCNPPaymentInformation(
 export function editCNPPaymentInformationToggled() {
   return { type: CNP_PAYMENT_INFORMATION_EDIT_TOGGLED };
 }
+
+export function fetchEDUPaymentInformation(recordEvent = recordAnalyticsEvent) {
+  return async dispatch => {
+    dispatch({ type: EDU_PAYMENT_INFORMATION_FETCH_STARTED });
+
+    recordEvent({ event: 'profile-get-edu-direct-deposit-started' });
+    const response = await getData('/profile/ch33_bank_accounts');
+
+    if (response.error) {
+      recordEvent({ event: 'profile-get-edu-direct-deposit-failed' });
+      dispatch({
+        type: EDU_PAYMENT_INFORMATION_FETCH_FAILED,
+        response,
+      });
+    } else {
+      recordEvent({
+        event: 'profile-get-edu-direct-deposit-retrieved',
```

Comment:
Maybe we should add a not here to make sure that when we can determine whether the user is eligible for DD for EDU, we also log whether they are, just like DD for Comp and Pen?